manifests/UPDATING-CHARTS.md

```diff
@@ -0,0 +1,58 @@
+# Upating charts and values.yaml
+
+The charts in the `manifests` directory are used in istioctl to generate an installation manifest. The configuration 
+settings contained in values.yaml files and passed through the CLI are validated against a
+[schema](../operator/pkg/apis/istio/v1alpha1/values_types.proto).
+Whenever making changes in the charts, it's important to follow the below steps.
+
+### Step 0. Check that the change really belongs in values.yaml
+
+Is this a new parameter being added? It so it probably doesn't belong in values.yaml. 
+Dynamic, runtime config that is used to configure Istio components should go into the
+[MeshConfig API](https://github.com/istio/api/mesh/v1alpha1/config.proto). Values.yaml is being deprecated and adding
+to it is discouraged. MeshConfig is a real, official API which follows API management practices and is dynamic
+(does not require component restarts).
+
+### Step 1. Make changes in charts and values.yaml in `manifests` directory
+
+If this is an edit to existing charts or values that doesn't change the schema, make your changes in the `manifests`
+directory first.
+
+### Step 2. Make corresponding values changes in [operator/data/profiles/default.yaml](../operator/data/profiles/default.yaml)
+
+The values.yaml in `manifests` are only used for direct Helm based installations, which is being deprecated. 
+If any values.yaml changes are being made, the same changes must be made in the `operator/data/profiles/default.yaml`
+file, which must be in sync with the Helm values in `manifests`.
+
+### Step 3. Update the validation schema
+
+Istioctl uses a [schema](../operator/pkg/apis/istio/v1alpha1/values_types.proto) to validate the values. Any changes to
+the schema must be added here, otherwise istioctl users will see errors. 
+Once the schema file is updated, run:
+```bash
+$ make gen-operator-proto
```

Comment:
maybe suggest build_with_container=1 until we get that as the default

Reply:
i was thinking the make command would include that. this is a target that it's never really possible to do a local build.

Reply:
Done.